lib/std/fs/path.zig

```diff
@@ -1271,11 +1271,12 @@ fn testRelativeWindows(from: []const u8, to: []const u8, expected_output: []cons
 
 /// Returns the extension of the file name (if any).
 /// This function will search for the file extension (separated by a `.`) and will return the text after the `.`.
-/// Files that end with `.` are considered to have no extension, files that start with `.`
+/// Files that start or end with `.` are considered to have no extension.
```

Comment:
How about:

```
/// Files that end with `.` are considered to have no extension.
/// Files that start with `.` and have no other `.` in their file name are considered to have no extension.
```

Seems more precise/accurate considering the `.gitignore => ""`, `.image.png => ".png"` examples.

Reply:
Good idea. I prefer one sentence though. I changed the docstring and added the test case.